examples/wallet_with_pincode/wallet_with_pincode.v.py

```diff
@@ -0,0 +1,77 @@
+# Simple wallet with PIN protection
+
+# bad guys can not get money out of this contract-wallet
+# even if your secret key has been compromised
+# if they do not know the `secret`
+# (pre-image of the hash that is stored in contract).
+
+# Owners wallet.
+owner: public(address)
+
+# Hash of a secret number that allows user
+# to operate with funds from this contract.
+secret_hash: public(bytes32)
+
+# Assign owner to contract creator.
+def __init__():
+    self.owner = msg.sender
```

Comment:
`secret_hash` should be set here, as the first time you go to use `set_secret()` below it might not work properly.